includes/definitions/ibmtl.yaml

```diff
@@ -5,6 +5,8 @@ icon: generic
 over:
     - { graph: device_bits, text: Traffic }
 discovery:
-    - snmpget:
+    -
+      sysObjectId: .1.3.6.1.4.1.14851
```

Comment:
http://www.snmplink.org/cgi-bin/nd/m/Ent/I/IBM%20Corp/IBM/ibmProd/TotalStorage%20TS%20(Tape%20Storage)%20Systems/TS-3500%20-%20Tape%20Library/3584%20TotalStorage%20UltraScalable%20Tape%20Library/2009__09__23/SNIA-SML-rev1-21.mib

Indicates that's probably right.

Reply:
Close enough :)